src/Equinor.ProCoSys.IPO.Command/InvitationCommands/CancelInvitation/CancelInvitationCommand.cs

```diff
@@ -7,6 +7,6 @@ public class CancelInvitationCommand : IRequest<Result<string>>, IInvitationComm
     {
         public CancelInvitationCommand(int invitationId) => InvitationId = invitationId;
 
-        public int InvitationId { get; init; }
+        public int InvitationId { get; }
```

Comment:
@hbjorgo , was this necessary or is it ok to remove?

Reply:
@hbjorgo

Reply:
It can be removed. Just double check by compiling after removal.